compair/static/modules/assignment/assignment-module_spec.js

```diff
@@ -62,6 +62,7 @@ describe('assignment-module', function () {
         "name": "Test Course",
         "year": 2015,
         "term": "Winter",
+        "sandbox": 0,
```

Comment:
please use `false` instead of `0` for the specs (helps to keep track of what type things are